### PR TITLE
OLH-1662 add CTA if the user only has one type of MFA

### PR DESCRIPTION
**Pull request by saralk**
### What changed

This adds a CTA to encourage the user to set up a secondary backup method for 2FA.

### Why did it change

To encourage adoption of backup methods

### Environment variables or secrets
- [ ] No environment variables or secrets were added or changed

- [X] Application configuration is up-to-date
- [ ] Documented in the README
- [X] Added to deployment steps
- [X] Added to local startup config

## Testing

Tested Locally:

Screenshot 2024-04-03 at 16 38 18

Screenshot 2024-04-03 at 16 38 33

### CODE REVIEW

File: src/components/security/security-controller.ts [hunk 1]
```diff
@@ -59,11 +59,21 @@ export async function securityGet(req: Request, res: Response): Promise<void> {
       })
     : [];
 
+  const hasAuthAppMethod = req.session.mfaMethods?.some((method) => {
+    return method.mfaMethodType === "AUTH_APP";
+  });
```
Comment (saralk) [inline src/components/security/security-controller.ts:64]: Do we need this? I added this in, but based on the wording of the message, I think we would always want to show the message when the user only has one method

File: src/locales/en/translation.json
Comment (saralk) [inline src/locales/en/translation.json:1]: we need welsh translations

File: src/components/security/security-controller.ts (hunk 1 above)
Comment (peterfajemisincabinetoffice) [reply src/components/security/security-controller.ts:64]: @saralk , do you still need an answer to this?

Comment (saralk) [reply src/components/security/security-controller.ts:64]: https://gds.slack.com/archives/C02F70D2LQN/p1712149445312499

**Review by peterfajemisincabinetoffice — APPROVED** [review]

**Pull request merged**